AllReadyApp/Web-App/AllReady/Controllers/ManageController.cs

```diff
@@ -176,6 +177,19 @@ public IActionResult AddPhoneNumber()
             return RedirectToAction(nameof(VerifyPhoneNumber), new { PhoneNumber = model.PhoneNumber });
         }
 
+        // POST: /Account/ResendPhoneNumberConfirmation
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        public async Task<IActionResult> ResendPhoneNumberConfirmation(string phoneNumber)
+        {
+            var user = GetCurrentUser();
+            var code = await _userManager.GenerateChangePhoneNumberTokenAsync(user, phoneNumber);
+
+            await _smsSender.SendSmsAsync(phoneNumber, "Your security code is: " + code);

```

Comment:
Should the message include the name of the app?  Something like "Your allReady account security code is: "